docs/src/pages/comparing-astro-vs-other-tools.md

```diff
@@ -0,0 +1,136 @@
+
+
+## Eleventy vs. Astro
+
+[Eleventy](https://www.11ty.dev/) is a popular static site builder, powered by Node.js. It is very similar to Astro, and conceptually aligned with Astro's "minimize client-side JavaScript" approach to web development.
+
+Eleventy supports several older [templating languages](https://www.11ty.dev/docs/languages/) for HTML: Nunjucks, Liquid, Pug, EJS, and others. Astro lets you write HTML using modern UI component libraries (React, Preact, Vue, Svelte) or a custom component syntax that is similar to HTML + JSX. Eleventy does not currently support using modern UI component libraries for HTML templating.
+
+Eleventy offers "zero client-side JavaScript" by not including any JavaScript on your site by default. By making JavaScript opt-in, Eleventy sites can be written with little to no JavaScript. However, when you do need client-side JavaScript, it's up to you to set up your own build pipeline. This can be time consuming, and is only as optimized as what you can build yourself with Webpack, Snowpack, or Vite.
+
+By contrast, Astro has built-in support for Partial Hydration and Island Architecture to only load the bare minimum amount of JavaScript in the browser. In production, Astro automatically optimizes your included component JavaSript bundles for you. This gives you built-in support to include individual JavaScript components on the page, without any effort on your part.
+
+#### Comparing Eleventy vs. Astro Performance
+
+Eleventy has few opinions about what you build, so it's difficult to compare generally. Because Eleventy includes zero JavaScript by default, it is possible for an Eleventy website to be as fast as an Astro website. However, because you are responsible for building your own build pipeline for any JavaScript that you include on the page, most Eleventy websites will not be as well-optimized as Astro websites.
+
+
+<!-- 
+## Gatsby vs. Astro
+
+**Next.js** is a popular website & application framework for React.
+
+
+## Hexo vs. Astro
+
+**Hexo** is a popular static site generator, powered by Node.js.
+
+-->
+
+## Hugo vs. Astro
+
+[Hugo](https://gohugo.io/) is a popular static site generator, powered by Go. 
+
+Hugo supports an custom [templating languages](https://gohugo.io/templates/introduction/) for HTML. Astro lets you write HTML using modern UI component libraries (React, Preact, Vue, Svelte) or a custom component syntax that is similar to HTML + JSX. Hugo does not support using modern UI component libraries for HTML templating.
```

Comment:
```suggestion
Hugo supports an custom [templating language](https://gohugo.io/templates/introduction/) for HTML. Astro lets you write HTML using modern UI component libraries (React, Preact, Vue, Svelte) or a custom component syntax that is similar to HTML + JSX. Hugo does not support using modern UI component libraries for HTML templating.
```

Reply:
I think it should also be "**a** custom [templating ...", or am I crazy?

Reply:
that too :0 i totally missed that

Reply:
thanks! fixed